wordpress-mobile/WordPress-iOS
# Site Creation: Create Site step

Fixes #8431

To test:
- Go through the new Create Site flow.
- On the Domain Suggestion page, select a domain and click 'Create site'.
- The next view (actually creating the site) will show the steps, and the text type will change once the step has been reached.
- The step labels correspond to the following backend steps:
  - Laying site foundation > validating site
  - Retrieving site information > creating site
  - Configure site content > setting tagline
  - Configure site style > setting theme
  - Preparing frontend > syncing site & account information

Once the site has been created, the following temporary alert is displayed. The 'Go Home' button dismisses the Site Creation Navigation Controller. You should then be able to navigate to My Sites and locate the new site.
<img width="275" alt="temp_alert" src="https://user-images.githubusercontent.com/1816888/35239146-dfd84232-ff6c-11e7-92fb-4c7333fa8aba.png">



After site creation, verify the following:
- Site name.
- Site domain.
- Site tagline.
- Site theme.


Also, verify the existing Signup process still works (signup + create site combo) by disabling the socialSignup feature flag.


**As designed:**
<img width="200" alt="designed" src="https://user-images.githubusercontent.com/1816888/35239202-0f9e0344-ff6d-11e7-9b3e-afcfa741a05e.png">



**Implemented:** 
![create_site](https://user-images.githubusercontent.com/1816888/35239058-9a70e38e-ff6c-11e7-8f64-96c7f1a7b8dc.gif)
